src/models/Weather/weather.typse.ts

```diff
@@ -1,4 +1,4 @@
-export type WeatherData = {
+export type Weather = {
```

Comment:
You have typo in filename: weather.typSe.ts

Reply:
fixed: renamed the file and corrected all its imports